### ISSUE TITLE
Installation failed: k is not defined

### ISSUE THREAD

**Issue by RikkiGibson** [shemetz/precise-drawing-tools#2]
I'm running Foundry 9.266. When I try to install the module using either the manifest URL or hitting "Install" in the package list, I get an error notification saying *Installation failed: k is not defined*. I dug around in the console and found this stack which looked relevant:

```
ReferenceError: k is not defined
    at Function.getManifest (file:///home/foundry/resources/app/dist/packages/package.mjs:1:7288)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:95:5)
    at async checkPackage (file:///home/foundry/resources/app/dist/packages/views.mjs:1:1295)
    at async Module.installPackage (file:///home/foundry/resources/app/dist/packages/views.mjs:1:3053)
    at async SetupView.handlePost (file:///home/foundry/resources/app/dist/server/views/setup.mjs:1:1757)
```

Maybe the error is happening on the server side?

**Comment by shemetz** [comment]
Fixed in "v1.0.0-b", I hope it works, let me know?

**Comment by RikkiGibson** [comment]
It works now, thank you so much!
